poller/README.md

```diff
@@ -56,30 +60,55 @@ instructions on how to change the payload.
-`scalingMethod`          | `STEPWISE`     | Scaling method that should be used. Options are: `STEPWISE`, `LINEAR`, `DIRECT`. <br /> See the [scaling methods section][autoscaler-scaler-methods] in the Scaler function page for more information.
+`scalingMethod`          | `STEPWISE`     | Scaling method that should be used. Options are: `STEPWISE`, `LINEAR`, `DIRECT`. See the [scaling methods section][autoscaler-scaler-methods] in the Scaler function page for more information.
 `stepSize`               | 2              | Number of nodes that should be added or removed when scaling with the `STEPWISE` method.
 `overloadStepSize`       | 5              | Number of nodes that should be added when the Cloud Spanner instance is overloaded, and the `STEPWISE` method is used.
 `scaleOutCoolingMinutes` | 5              | Minutes to wait after scaling IN or OUT before a scale OUT event can be processed.
 `scaleInCoolingMinutes`  | 30             | Minutes to wait after scaling IN or OUT before a scale IN event can be processed.
-`overloadCoolingMinutes` | 5              | Minutes to wait after scaling IN or OUT before a scale OUT event can be processed, when the Spanner instance is overloaded.<br/> An instance is overloaded if its High Priority CPU utilization is over 90%.
-`stateProjectId`         | `${projectId}` | The project ID where the Autoscaler state will be persisted. <br /> By default it is persisted using [Cloud Firestore][cloud-firestore] in the same project as the Spanner instance.
+`overloadCoolingMinutes` | 5              | Minutes to wait after scaling IN or OUT before a scale OUT event can be processed, when the Spanner instance is overloaded. An instance is overloaded if its High Priority CPU utilization is over 90%.
+`stateProjectId`         | `${projectId}` | The project ID where the Autoscaler state will be persisted. By default it is persisted using [Cloud Firestore][cloud-firestore] in the same project as the Spanner instance.
+`metrics`                | Array          | Array of objects that can override the values in the metrics used to decide when the Cloud Spanner instance should be scaled IN or OUT. Refer to the [metrics definition table](#metrics-parameters) to see the fields used for defining metrics.
 
-## Custom Thresholds
+## Metrics parameters
+
+The table describes the objects used to define metrics. These can be provided
+in the configuration objects to customize the metrics used to autoscale your
+Cloud Spanner instances.
+
+To specify a custom threshold specify the name of the metrics to customize
+followed by the parameter values you wish to change. The updated parameters
+will be merged with the default metric parameters.
+
+### Selectors
+
+Key                  | Description
+---------------------| -----------
+`name`               | The unique name of the for the metric to be evaulated. The default metrics are `high_priority_cpu`, `rolling_24_hr` and `storage`.
```

Comment:
"**A** unique name ..."
It should be clear that this is a just a unique identifier that one can use for the metric. 

Naming the default metrics should be in the context of:
"In you want to override the default metrics, their names are: ...."

Reply:
This implementation is pretty close to supporting custom metrics.  I think there are a couple things we should do in addition to this PR for custom metrics, added some thoughts to the Issue #10 around additional changes for custom metrics.

Reply:
README updated, per your recommendations.